bundles/binding/org.openhab.binding.xbmc/src/main/java/org/openhab/binding/xbmc/rpc/calls/PlayerGetItem.java

```diff
@@ -23,6 +23,8 @@
  *
  * @author tlan, Ben Jones, Marcel Erkel
  * @since 1.5.0
+ * @author tlan, Ben Jones, Plebs
+ * @since 1.9.0
```

Comment:
Just add your name  to line 24 (and add back Marcel Erkel), and leave it at 1.5.0, and remove lines 26 & 27.